src/app/components/Welcome/Welcome.jsx

```diff
@@ -44,11 +44,20 @@ class Welcome extends React.Component {
         },
         connector: "lnd",
       };
-      console.log(this.settings.salt);
-      account.config = encryptData(account.config, "btc", this.settings.salt);
-      return this.accounts.setAccount(account, true).then(() => {
-        alert('Test account is saved. Your password is: btc');
-      });
+      return utils
+        .call("accounts.unlock", { password: "btc" })
+        .then((unlocked) => {
+          console.log(unlocked);
+          return utils.call("accounts.add", account).then((added) => {
+            console.log(added);
+            return utils
+              .call("accounts.select", { accountId: added.accountId })
+              .then((selected) => {
+                console.log(selected);
+                alert("Test account is saved. Your password is: btc");
+              });
```

Comment:
The UI always calls the backend script. So here first we `unlock` the extension (set the password) then we add an account (the password from the state is used to encrypt the config) and then we select the newly created account.